src/SimpleSimulator.cpp

```diff
@@ -547,6 +550,10 @@ int main(int argc, char **argv) {
     std::cerr << "Prefetching switch off?: " << vm["prefetch-off"].as<bool>() << std::endl;
     SimpleSimulator::prefetching_on = !(vm["prefetch-off"].as<bool>());
 
+    // Flag to turn on shuffling of jobs
+    std::cerr << "Job shuffling on?: " << vm["shuffle-jobs"].as<bool>() << std::endl;
```

Comment:
Maybe we should start to think of a verbosity level with all the new parameters we recently included.